xcdat/regridder/xgcm.py

```diff
@@ -82,27 +83,26 @@ def __init__(
         Import xCDAT:
 
         >>> import xcdat
-        >>> from xcdat.regridder import xgcm
 
         Open a dataset:
 
-        >>> ds = xcdat.open_dataset("so.nc")
+        >>> ds = xcdat.open_dataset("...")
 
         Create output grid:
 
         >>> output_grid = xcdat.create_grid(lev=np.linspace(1000, 1, 5))
 
-        Create theta:
+        Regrid data to ``output_grid``:
 
-        >>> ds["pressure"] = (ds["hyam"] * ds["P0"] + ds["hybm"] * ds["PS"]).transpose(**ds["T"].dims)
+        >>> output_data = ds.regridder.vertical("so", output_grid, tool="xgcm", method="linear")
 
-        Create regridder:
+        Create pressure variable:
 
-        >>> regridder = xgcm.XGCMRegridder(ds, output_grid, method="linear", target_data="pressure")
+        >>> ds["pressure"] = (ds["hyam"] * ds["P0"] + ds["hybm"] * ds["PS"]).transpose(**ds["T"].dims)
 
-        Regrid data:
+        Regrid data to ``output_grid`` in pressure space:
 
-        >>> data_new_grid = regridder.vertical("so", ds)
+        >>> output_data = ds.regridder.vertical("so", output_grid, tool="xgcm", method="linear", target_data="pressure")
 
         Passing additional arguments to ``xgcm.Grid`` and ``xgcm.Grid.transform``:
```

Comment:
I can't suggest the edit for the example below because it is unchanged, but it should look like this:

```python
        >>> regridder = xgcm.XGCMRegridder(
        >>>     ds,
        >>>     output_grid,
        >>>     method="linear",
        >>>     extra_init_options={"boundary": "fill", "fill_value": 1e27},
        >>>     mask_edges=True,
        >>> )
```

Reply:
Fixed with https://github.com/xCDAT/xcdat/pull/535/commits/8828bc80f12626ad623a546145288a99ddb1a26f.